package.json

```diff
@@ -65,44 +65,44 @@
     "@bbc/moment-timezone-include": "^1.1.4",
     "@bbc/psammead-amp-geo": "^1.1.2",
     "@bbc/psammead-assets": "^2.14.0",
-    "@bbc/psammead-brand": "^5.1.15",
-    "@bbc/psammead-bulleted-list": "1.0.7",
-    "@bbc/psammead-bulletin": "^3.0.1",
```

Comment:
Just wondering whether the removal of the carat here was intentional?

Reply:
it was. I bumped `psammead-bulletin` to version 3.0.5, however that package uses psammead-story-promo@6.x.x, which is not yet pulled into Simorgh, so that caused some duplication errors.

Not sure if `psammead-story-promo` is ready to be bumped to v6, so I felt it was safer to leave this bump out of this PR.

Reply:
Ok. Thanks for the explanation.

Reply:
This has now been updated. `psammead-story-promo` is being update to v6, so we can also pull in latest `psammead-bulletin` version